### ISSUE TITLE
breaks if final semi-colon not included

### ISSUE THREAD

**Issue by mildfuzz** [AndreasBackx/StyleSorter#7]
If the last semi-colon of a rule set it omitted, as is allowed in CSS, the rule does not copy correctly.

**Comment by AndreasBackx** [comment]
This will be fixed in the lexer version of this plugin. I'll start working on this when I have time. When I'm done with other projects or have some spare time on my hands.

**Comment by AndreasBackx** [comment]
After having a look at lexers, checking whether the stylesheet is valid is not gonna be implemented. When you sort your stylesheet you are sure that there are no problems in your CSS.

**Comment by mildfuzz** [comment]
It is valid to skip the final semicolon of a rule block.

**Comment by AndreasBackx** [comment]
@mildfuzz You're right, I'll make sure this get added in the upcoming lexer version.
